packages/next-swc/crates/napi/build.rs

```diff
@@ -1,5 +1,23 @@
+use std::{
+    env,
+    fs::File,
+    io::{BufWriter, Write},
+    path::Path,
+};
+
 extern crate napi_build;
 
 fn main() {
+    let out_dir = env::var("OUT_DIR").expect("Outdir should exist");
```

Comment:
Is this being set right now? Would it break the napi-rs builds?

Reply:
Seems like it is being set by napi-rs, is that correct?

Reply:
no, it's set by cargo (https://doc.rust-lang.org/cargo/reference/environment-variables.html), as well as other env var in this 
build script (`TARGET`).

> OUT_DIR — If the package has a build script, this is set to the folder where the build script should place its output

> TARGET — the target triple that is being compiled for. Native code should be compiled for this triple.

These env var should be set as long as build goes through cargo (napi-rs cli wraps cargo as well). If someone uses rustc directly to build package it'll cause compilation error, which is highly unlikely in next-swc's case.